epimodel/exports/epidemics_org.py

```diff
@@ -287,8 +287,7 @@ def assert_valid_json(file, minify=False):
             )
 
 
-def upload_export(dir_to_export, config, channel="test"):
-    """The 'upload' subcommand"""
+def upload_export(dir_to_export, gs_prefix, channel: str):
```

Comment:
Here I was the most unsure by the changes I did. @mathijshenquet I may need some consulting here